doc/COMPILING/COMPILING.md

```diff
@@ -191,45 +191,50 @@ Run:
+    mkdir -p ~/src/Cataclysm-DDA
     mkdir -p ~/src/mxe
-    git clone https://github.com/mxe/mxe.git ~/src/mxe
-    cd ~/src/mxe
-    make MXE_TARGETS='x86_64-w64-mingw32.static i686-w64-mingw32.static' sdl2 sdl2_ttf sdl2_image sdl2_mixer gettext ncurses
+    cd ~/src
+    git clone https://github.com/CleverRaven/Cataclysm-DDA.git ./Cataclysm-DDA
+    git clone https://github.com/mxe/mxe.git ./mxe
+    make -j$((`nproc`+0)) MXE_TARGETS='x86_64-w64-mingw32.static i686-w64-mingw32.static' sdl2 sdl2_ttf sdl2_image sdl2_mixer gettext
 
-If you are not on a Debian derivative (Linux Mint, Ubuntu, etc), you will have to use a different command than apt-get to install [the MXE requirements](http://mxe.cc/#requirements). Building all these packages from MXE might take a while even on a fast computer. Be patient. If you are not planning on building for both 32-bit and 64-bit, you might want to adjust your MXE_TARGETS.
+Building all these packages from MXE might take a while, even on a fast computer. Be patient; the `-j` flag will take advantage of all your processor cores. If you are not planning on building for both 32-bit and 64-bit, you might want to adjust your MXE_TARGETS.
 
-### Building (SDL)
+Edit your `~/.profile` as follows:
 
-Run:
+    export PLATFORM_32="~/src/mxe/usr/bin/i686-w64-mingw32.static-"
+    export PLATFORM_64="~/src/mxe/usr/bin/x86_64-w64-mingw32.static-"
 
-    PLATFORM="i686-w64-mingw32.static"
-    make CROSS="~/src/mxe/usr/bin/${PLATFORM}-" TILES=1 SOUND=1 RELEASE=1 LOCALIZE=1
+This is to ensure that the variables for the `make` command will not get reset after a power cycle.
 
-Change PLATFORM to `x86_64-w64-mingw32.static` for a 64-bit Windows build.
+### Building (SDL)
 
-To create a nice zip file with all the required resources for a trouble free copy on Windows use the bindist target like this:
+    cd ~/src/Cataclysm-DDA
 
-    PLATFORM="i686-w64-mingw32.static"
-    make CROSS="~/src/mxe/usr/bin/${PLATFORM}-" TILES=1 SOUND=1 RELEASE=1 LOCALIZE=1 bindist
+***IMPORTANT:***
 
-### Building (ncurses)
+The first time you set up your build environment, you must `touch VERSION.txt` to create a dummy file to avoid `make` complaining about not having a rule. You will need to add "VERSION.txt" to /.git/info/exclude in order to prevent your system from trying to `git push` this dummy file. Subsequent builds should not require `touch` again.
 
-Run:
+Run one of the following commands based on your targeted environment:
+
+    make -j$((`nproc`+0)) CROSS="${PLATFORM_32}" TILES=1 SOUND=1 RELEASE=1 LOCALIZE=1 BACKTRACE=0 PCH=0 bindist
+    make -j$((`nproc`+0)) CROSS="${PLATFORM_64}" TILES=1 SOUND=1 RELEASE=1 LOCALIZE=1 BACKTRACE=0 PCH=0 bindist
 
-    PLATFORM="i686-w64-mingw32.static"
-    make CROSS="~/src/mxe/usr/bin/${PLATFORM}-" RELEASE=1 LOCALIZE=1
+
+<!-- Building ncurses for Windows is a nonstarter, so the directions were removed. -->
```

Comment:
I was able to build the ncurses version on Windows with MSYS2, so it is still possible if the cross toolchain has the suitable libs.

Reply:
Oh, that's good to know, I suppose. However, per Kevin:

> [3:12 AM] ToxiClay: Is it OK if I strip ncurses out of compilation instructions for Windows, as we won't be compiling ncurses builds on or for Windows? Like, the Cygwin instructions call for apt-cyg install libncurses-devel, and I'm pretty sure I can remove it with no problem
> 
> [3:14 AM] kevingranade: yes

Reply:
I'm kinda surprised you were able to build it; I was under the impression Windows ncurses simply didn't exist.

Reply:
It might work, but it's not at all supported, and it's subject to being ripped out by the roots.
The rationale for which is that on windows our curses replacement library has worse performance than the SDL build in text mode, and as far as I know no features are enabled by the terminal build on windows.
Specifically, on linux having a terminal build enables playing over ssh and easier integration with some screen readers, I'm not aware of any such use cases on windows.

Reply:
By ncurses version I mean the original linux ncurses ported to Windows, provided by MSYS2, not our curses 'port' implemented using the Windows API. I managed to build it when I was doing UI fixes and wanted to test if it worked in the ncurses version but didn't have a linux machine. I imagine with the proper ported libs and tools, playing over ssh and with screen readers could be possible. That said, I don't think anyone would want to build on Windows with ncurses apart from testing, so it's probably fine not to mention it in the docs.

Reply:
oh wow, I had absolutely no idea that was functional.